parser/ast/functions.go

```diff
@@ -300,6 +300,7 @@ const (
 	TiDBShard       = "tidb_shard"
 	GetLock         = "get_lock"
 	ReleaseLock     = "release_lock"
+	Grouping        = "grouping"
```

Comment:
shoul not add this since we actually not support it yet.

Reply:
> shoul not add this since we actually not support it yet.

I will delete it.

Reply:
> shoul not add this since we actually not support it yet.

UT need this field, may be we have to keep it.